### PR TITLE
build: remove conventional changelog

### PR DESCRIPTION
**Pull request by ekashida**
## Details

Follow up to our discussion to work on better release notes. Also bumps the commit subject to a max length of 100.

## Does this PR introduce breaking changes?

* ✅ `No, it does not introduce breaking changes.`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,16 +17,13 @@
     "test:ci": "yarn test --no-cache --coverage --runInBand",
     "test:integration": "lerna exec --scope integration-tests -- yarn sauce",
     "test:performance": "lerna exec --scope perf-benchmarks -- best --runner remote",
-    "changelog:generate": "conventional-changelog -p angular -i CHANGELOG.md -s -r 0",
     "release:ci:npm": "./scripts/release_npm.sh",
-    "release:ci:changelog": "./scripts/release_changelog.sh",
```
Comment (ekashida) [inline package.json]: Do we need to update our CI so that it does not invoke this?

**Comment by diervo** [comment]
Why? I really like the enforcement (altough Eugene was the only one following it )

**Comment by ekashida** [comment]
@diervo The linting is still there, this just removes the changelog generation.

File: package.json (hunk 1 above)
Comment (apapko) [reply package.json]: I believe so

**Comment by pmdartus** [comment]
@apapko I just want to double-check since you created this flow in the first place. Do you still have concerns with the new approach to manually write the release notes?

File: package.json (hunk 1 above)
Comment (ekashida) [reply package.json]: Done

**Pull request merged**